libiocage/lib/Jail.py

```diff
@@ -401,6 +424,12 @@ def _force_stop(self):
         successful = True
 
         try:
+            self._run_hook("prestop")
+        except:
+            successful = False
```

Comment:
I can't see in this diff where successful is used. Is it?

Reply:
Yes, it is. Right at the end of the method